crdsonnet/main.libsonnet

```diff
@@ -63,22 +63,22 @@ local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
       local renderEngine = _processor.renderEngine;
       local grouping = helpers.getGroupKey(definition.spec.group, groupSuffix);
       local name = helpers.camelCaseKind(this.getKind(definition));
-      std.foldl(
-        function(acc, version)
-          local schema = this.getSchemaForVersion(definition, version);
-          acc
-          + renderEngine.toObject(
-            renderEngine.nestInParents(
+      renderEngine.toObject(
```

Comment:
This change (and simliar) are resolving an issue with nilvalue:

Each engine operates on its own internal representation, for `dynamic` that is an object (with fields obviously), for `static` that is a string (a string representation of fields) and for `ast` that is an array (of fields).

When calling `render()`, `toObject` is called to ensure delivering an object, as `dynamic` operates on objects, this function will simply returns the object, `static` wrap the string representation in curly brackets `{}` and finaly `ast` will put it in an astObject (`j.object.member`).

Functions in `main.libsonnet` that operate on the output of `render()` (to append additional functions or nest into other objects), also need to call `toObject` on this new output, including the `nilvalue`.